extensions/blocks/business-hours/business-hours.php

```diff
@@ -119,7 +119,7 @@ function jetpack_business_hours_render( $attributes ) {
 			}
 			$days_hours .= sprintf(
 				/* Translators: Business opening hours info. */
-				_x( 'From %1$s to %2$s', 'from business opening hour to closing hour', 'jetpack' ),
+				'%1$s - %2$s',
```

Comment:
Note leftover "translators" comment above this line.

Reply:
Well spotted! Fixed in b3b1bfc